splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/export/ExportFileTest.java

```diff
@@ -111,7 +111,8 @@ public void createDirectory_returnsFalseWhenCannotCreate() throws IOException, S
             assertFalse(exportFile.createDirectory());
         }
         catch (Exception e) {
-             Assert.assertTrue(e.getMessage(), e.getMessage().contains("IOException '/noPermissionToCreateFolderInRoot' when accessing directory"));
```

Comment:
That's a good use case for hamcrest: more concise, more readable and out of the box comprehensive error message:
`assertThat(e.getMessage(), anyOf(containsString(strA), containsString(strB))`
or, if you're adventurous:
`assertThat(e.getMessage(), matchesPattern("IOException '/noPermissionToCreateFolderInRoot(: Read-only file system)+' when accessing directory"))`
The second one might need adjustments, I'm a bit rusty on patterns.

Reply:
done, thanks for the suggestion.